### ISSUE TITLE
False positive warning of ExecNotFound

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/systemdlint#39]
```
rootfs/etc/systemd/system/sysinit.target.wants/apparmor.service :18:1 - Command referenced not found - [systemdlint.systemdlint.ExecNotFound]
```

with the unit having set 

```
ExecStop=/sbin/apparmor_parser --remove /etc/apparmor.d/
```

and rootfs having

```
-rwxr-xr-x 1 user group 1344624 Mar 20 13:32 sbin/apparmor_parser
```
